src/features/duckplayer/text.js

```diff
@@ -8,15 +10,30 @@ const text = {
     videoOverlayTitle: {
         title: 'Tired of targeted YouTube ads and recommendations?'
     },
+    videoOverlayTitle_a1: {
+        title: 'Turn on Duck Player to watch without targeted ads'
+    },
+    videoOverlayTitle_b1: {
+        title: 'Drowning in ads in YouTube? {newline} Turn on Duck Player.'
+    },
     videoOverlaySubtitle: {
         title: 'provides a clean viewing experience without personalized ads and prevents viewing activity from influencing your YouTube recommendations.'
     },
+    videoOverlaySubtitle_a1_b1: {
+        title: 'What you watch in DuckDuckGo won’t influence your recommendations on YouTube.'
+    },
```

Comment:
is this key `videoOverlaySubtitle_a1_b1` created to indicate that this string is used in 2 places in the app? if so I don't think we should allow experiment details leak into this file. This file is just a bunch of strings that all need unique identifiers - any mappings can be done in your code.

Reply:
Makes sense. I’ve updated the translation string ids